src/foundry/client-esm/applications/_types.d.mts

```diff
@@ -2,6 +2,8 @@
 
 import type ApplicationV2 from "./api/application.d.mts";
 
+export {};
```

Comment:
I noticed that types from this file were still getting imported. My intent were for these to be fully internal details so that users didn't accidentally import it. I suppose my point was proven by the fact that these started getting imported already.

It must be if your only imports are `import type` then it still can be imported but _is_ able to get access to ambient declarations unlike if you were dealing with 0 imports/exports. That's pretty strange, to be honest.